source/reference/collection-level-filtering.txt

```diff
@@ -42,31 +98,46 @@ Configure a Filter
 
    .. step:: Create a Filter.
 
-      The ``includeNamespaces`` parameter specifies an optional filter
-      that you can pass to the :ref:`/start <c2c-api-start>` API.
+      The :ref:`c2c-api-start` API accepts two parameters that configure
+      optional filters: 
       
-      If you don't specify a filter, ``mongosync`` does a full cluster
+      - The ``includeNamespaces`` parameter takes an array of filters, which
+        are used to determines which databases and collections ``mongosync`` 
+        should include in the sync
+      - The ``excludeNamespaces`` parameter takes an array of filters, which
+        are used to determine which databases and collections ``mongosync``
+        should exclude from the sync.
+     
+      If you don't specify a filter, ``mongosync`` performs a full cluster
       sync.
 
-      The filter syntax is: 
+      Create inclusion or exclusion filters to identify the databases and
```

Comment:
Should we say `and / or` exclusion filters?

Reply:
Fixed